packages/core/src/config.ts

```diff
@@ -637,6 +664,14 @@ const composeExternalsConfig = (
           },
         },
       };
+    case 'mf':
+      return {
+        output: externals
+          ? {
+              externals,
+            }
+          : {},
```

Comment:
We could add `"mf": "var"` (default in Rspack/webpack) to `externalsTypeMap` above, and it could indicate which external type does MF will apply. By the way, does `MF` always use `"var"` external type?

Reply:
no mandatory requirement to use which external type